auto.def

```diff
@@ -123,7 +127,7 @@ options {
 if {1} {
   # Keep sorted, please.
   foreach opt {
-    autocrypt backtrace bdb coverage doc everything fmemopen full-doc gdbm
+    autocrypt bdb coverage debug-backtrace debug-graphviz debug-notify debug-parse-test debug-window doc everything fmemopen full-doc gdbm
```

Comment:
I'd love to keep the line lengths reasonable

Reply:
Folded